### ISSUE TITLE
Rampage Knights

### ISSUE THREAD

**Issue by bxrninvxid** [skyline-emu/skyline-games-list#1381]
**Is there an existing issue for this?**
 
I have searched the existing issues

**Tested Build**
1793
**Android Version**
13

**SoC type**
Snapdragon 865

**Device GSMArena link**
[Oneplus 8
](https://www.gsmarena.com/oneplus_8-10161.php)

**Game Behaviour**
60 FPS
Sound work
No graphic errors
fully playable

![Screenshot_2023_01_26_04_06_17_57_fcf072b9e4efc4ce5ca3e36691a7c3c6](https://user-images.githubusercontent.com/42973027/214934075-ee7771ab-30aa-48d1-8f84-a234b59b5245.jpg)

**Is Log Level is set to 'Info'?**
 Yes

**Log file**
[RampageKnights.txt](https://github.com/skyline-emu/skyline-games-list/files/10512808/RampageKnights.txt)

**Comment by bxrninvxid** [comment]
> Can you finish game?

haven't managed to finish it yet, but I can get to first boss and then to arena (where i died last time) without any problems

**Comment by bxrninvxid** [comment]
> Can you finish game?

Yes. It's possible to finish the game with no issues. Here's final boss gameplay:
https://youtu.be/buAWa-5DMEk
(fps drops cuz of recording)

**Comment by Jovanking** [comment]
Wrete do I find rom for this game
